scripts/apps/authoring/metadata/metadata.ts

```diff
@@ -656,6 +656,13 @@ function MetaTermsDirective(metadata, $filter, $timeout, preferencesService, des
                 scope.uniqueField = value || 'qcode';
             });
 
+            // add scheme to terms
+            if (scope.list && scope.cv) {
+                angular.forEach(scope.list, (listItem) => {
```

Comment:
can you use [Array.prototype.forEach()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)?

Reply:
yes, I can, but to keep things consistent I need to change all `angular.forEach` in this file or at least in `MetaTermsDirective`, because `angular.forEach` is used there.

Reply:
I've changed it in all possible places in the current directive, except the ones where `undefined` might come, unlike  `Array.prototype.forEach()` the `angular.forEach` will not throw an error in this case.

Reply:
> yes, I can, but to keep things consistent I need to change all `angular.forEach` in this file

I don't think there's a need to update it all at once. It's fine if you did, but I don't see any issues if you left other instances as they were.

I would suggest updating it all if it was some complex piece of business logic, but it's a simple helper function that can be understood and refactored in 5 minutes so we wouldn't gain much by refactoring it all upfront.